### ISSUE TITLE
hard freeze

### ISSUE THREAD

**Issue by optimisticcynic** [EDCD/EDMarketConnector#457]
[EDMarketConnector.log](https://github.com/Marginal/EDMarketConnector/files/3665099/EDMarketConnector.log)
Since the update if I have elite mark dangerous market connector up and I open elite dangerous my computer freezes. cnt+alt+delete does not work and I need to hard reset my pc.

**Comment by Marginal** [comment]
Have you previously set "E:D journal file location" to a shared drive on another machine which is no longer available? 
If so, please try killing the app, running Regedit and deleting the value HKEY_CURRENT_USER\Software\Marginal\EDMarketConnector\journaldir

**Comment by optimisticcynic** [comment]
I have not changed my set up at all, and everything was saved locally. I did attempt to do your suggestion but it didn't fix the issue.

**Comment by gthubs** [comment]
I had the same problem: when EliteDangerous64.exe is running (no matter at what stage, even before loading the game menu) and I open EDMarketConnector (3.43 or 3.50b0) the system (Windows 10 x64) would freeze almost completely: the image on the monitor freezes (even the clock), the cursor doesn't move, the keyboard has zero reaction. The music from the ED continues to play, but I can't do anything except a hardware reboot.

In my case temporary solution was disabling the Comodo Firewall, or rather the HIPS module. After dozens of freezes I found out that the problem is present if in the "HIPS Rules" for the EDMarketConnector I set the rule "Interprocess Memory Accesses" -> "Action" = "Ask". If I select "Allow" or "Block" everything is fine. I want to note that I’ve been using Comodo for more than 7 years, but this is the first time I've met this behavior.

I don' know how to collect more information. The logs are empty, I tried to calling an BSOD via the Ctrl + Scroll function and save a memory dump, but the keyboard useless. I understand that the problem is very rare, however I’m ready to assist in the diagnostics / testing of patches, etc.
